core/src/main/resources/hudson/PluginManager/installed.jelly

```diff
@@ -70,7 +70,7 @@ THE SOFTWARE.
                 <th width="1">${%Uninstall}</th>
               </tr>
               <j:forEach var="p" items="${app.pluginManager.plugins}">
-                <tr class="plugin ${p.hasMandatoryDependents()?'has-dependents':''} ${(p.hasMandatoryDependents() and !p.enabled)?'has-dependents-but-disabled':''} ${p.isDeleted()?'deleted':''} ${p.detached and p.enabled ? 'detached' : ''}" data-plugin-id="${p.shortName}" data-plugin-name="${p.displayName}">
+                <tr class="plugin ${p.hasMandatoryDependents()?'has-dependents':''} ${(p.hasMandatoryDependents() and !p.enabled)?'has-dependents-but-disabled':''} ${p.isDeleted()?'deleted':''} ${p.hasImpliedDependents() and p.enabled ? 'detached' : ''}" data-plugin-id="${p.shortName}" data-plugin-name="${p.displayName}">
```

Comment:
Not sure I follow what this PR is doing at a user level, but from a code review perspective it is very confusing since it is leaving a CSS class `detached` which no longer corresponds to whether a plugin is in fact detached.

Reply:
> Not sure I follow what this PR is doing at a user level

For every plugin X, I check whether any plugin Y is installed that depends on a core older than the core in which X was detached. Only then can it be an implied dependency.

> it is very confusing since it is leaving a CSS class

Good point, fixing.